2025-10-28 13:36:07 | scripts.extraction_pipeline.extractor_core | INFO | üß© Logger initialized for PREPROCESSING ‚Üí /Users/kousiknandury/PycharmProjects/DL_project/logs/extraction_logs/preprocessing_20251028_133607.log
2025-10-28 13:36:14 | scripts.extraction_pipeline.extractor_core | INFO | üìÑ Extracting text from PDF: empty.pdf
2025-10-28 13:36:14 | scripts.extraction_pipeline.extractor_core | INFO | Detected 0 pages in empty.pdf
2025-10-28 13:36:14 | scripts.extraction_pipeline.extractor_core | INFO | ‚úÖ Extraction complete for empty.pdf | Pages: 0 | Time: 0.00s
2025-10-28 13:36:14 | scripts.extraction_pipeline.extractor_core | INFO | üìÑ Extracting text from PDF: corrupt.pdf
2025-10-28 13:36:14 | scripts.extraction_pipeline.extractor_core | ERROR | ‚ùå Extraction failed for corrupt.pdf: Corrupted file
Traceback (most recent call last):
  File "/Users/kousiknandury/PycharmProjects/DL_project/scripts/extraction_pipeline/extractor_core.py", line 37, in extract_text
    doc = fitz.open(file_path)
  File "/Users/kousiknandury/PycharmProjects/DL_project/tests/test_edge_cases.py", line 93, in <lambda>
    monkeypatch.setattr("fitz.open", lambda f: (_ for _ in ()).throw(RuntimeError("Corrupted file")))
  File "/Users/kousiknandury/PycharmProjects/DL_project/tests/test_edge_cases.py", line 93, in <genexpr>
    monkeypatch.setattr("fitz.open", lambda f: (_ for _ in ()).throw(RuntimeError("Corrupted file")))
RuntimeError: Corrupted file
2025-10-28 13:36:15 | scripts.extraction_pipeline.extractor_core | INFO | üñºÔ∏è Running OCR on image: loan_form.jpeg
2025-10-28 13:36:15 | scripts.extraction_pipeline.extractor_core | INFO | ‚úÖ OCR completed for image loan_form.jpeg in 0.00s
